src/SprykerEco/Zed/AdyenApi/Business/Converter/GetPaymentMethodsConverter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AdyenApi\Business\Converter;
+
+use Generated\Shared\Transfer\AdyenApiResponseTransfer;
+
+class GetPaymentMethodsConverter extends AbstractConverter implements AdyenApiConverterInterface
```

Comment:
I did it on purpose. I know it doesn't have any practical meaning but I want to see all interfaces that concrete class implements.

Reply:
You have 2 options here:
- remove interface from class, since it's useless there.
- remove abstract class and use DI  (my preferable solution, I hate abstract classes)